ops/model.py

```diff
@@ -258,12 +262,19 @@ def __init__(self, backend):
         self._backend = backend
         self._data = {}
 
-    def get(self, relation):
-        if not isinstance(relation, Relation):
-            raise ModelError('expected Relation instance, got {}'.format(type(relation).__name__))
-        binding = self._data.get(relation)
+    def get(self, binding_key):
+        if isinstance(binding_key, Relation):
+            binding_name = binding_key.name
+            relation_id = binding_key.id
+        elif isinstance(binding_key, str):
+            binding_name = binding_key
+            relation_id = None
+        else:
+            raise ModelError('a binding key must be a str or Relation, not {}'.format(type(binding_key).__name__))
```

Comment:
Suggested message tuning:

_'binding key must be str or relation instance, not {}'_

Reply:
https://github.com/canonical/operator/pull/157/commits/cea632893deb9afe259a2dab23d6f1fd12076e0e